src/lib/dom-api.html

```diff
@@ -593,11 +593,73 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        configurable: true,
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+
+          if (ownerDocument || !window.ShadowDOMPolyfill) {
+            return this.node.activeElement;
+          } else {
+            return window.ShadowDOMPolyfill.wrapIfNeeded(this.node).activeElement;
```

Comment:
This file has a wrap implementation so I think this can just be:

`return DomApi.wrap(this.node).activeElement;`